src/main/java/com/coherent/training/selenium/kapitsa/web/base/BrowserDriverFactory.java

```diff
@@ -21,22 +33,57 @@ public WebDriver createDriver(){
         switch (browser){
             case "chrome":
                 System.setProperty("webdriver.chrome.driver", CHROME_DRIVER);
-                driver.set(new ChromeDriver());
+                ChromeOptions chromeOptions = setChromeDownloadOptions();
+                driver.set(new ChromeDriver(chromeOptions));
             break;
             case "firefox":
                 System.setProperty("webdriver.gecko.driver", FIREFOX_DRIVER);
-                driver.set(new FirefoxDriver());
+                FirefoxOptions firefoxOptions = setFirefoxDownloadOptions();
+                driver.set(new FirefoxDriver(firefoxOptions));
                 break;
             case "edge":
                 System.setProperty("webdriver.edge.driver", EDGE_DRIVER);
-                driver.set(new EdgeDriver());
+                EdgeOptions edgeOptions = setEdgeDownloadOptions();
+                driver.set(new EdgeDriver(edgeOptions));
                 break;
             default:
                 System.setProperty("webdriver.chrome.driver", CHROME_DRIVER);
-                driver.set(new ChromeDriver());
+                ChromeOptions defChromeOptions = setChromeDownloadOptions();
+                driver.set(new ChromeDriver(defChromeOptions));
                 break;
         }
 
         return driver.get();
     }
+
+    private ChromeOptions setChromeDownloadOptions(){
+        prefs = new HashMap<>();
+        prefs.put("download.default_directory", PROJECT_DIR + File.separator + DOWNLOAD_FOLDER + File.separator + "chrome");
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.setExperimentalOption("prefs", prefs);
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setCapability(ChromeOptions.CAPABILITY, chromeOptions);
+
+        return chromeOptions;
+    }
+
+    private FirefoxOptions setFirefoxDownloadOptions(){
+        FirefoxProfile profile = new FirefoxProfile();
+        profile.setPreference("browser.download.folderList", 2);
+        profile.setPreference("browser.download.dir", PROJECT_DIR + File.separator + DOWNLOAD_FOLDER + File.separator + "firefox");
```

Comment:
try to check if `/` working for Windows as a line separator, I guess Java takes care of it